roadmap/frontend-development.md

```diff
@@ -0,0 +1,5 @@
+# Frontend Development
+A front-end developer is a person who develops the user interface of the website and applications. They are responsible for determining the structure and design of the web application, build features to improve user experience, make a balance between design and functionality, optimize web app for different devices, optimize pages for speed and scalability, use different markup languages for coding web pages, maintain brand consistency and write reusable codes. In simple terms, front-end development refers to the lines of code that determine the operation and structure of the user interface.
+
+## So here is the Roadmap:-
+<img src="https://drive.google.com/file/d/11Y6FtAoKQwe6DvXJIPx9ZMOG0r9FAW_i/view?usp=sharing">
```

Comment:
This image not working

Reply:
image issue resolve please merge this. @vinzvinci